ci-operator/config/kubevirt/kubevirt/kubevirt-kubevirt-master.yaml

```diff
@@ -4,22 +4,36 @@ base_images:
     name: "4.4"
     namespace: ocp
     tag: base
+  kubevirt-builder:
+    cluster: https://api.ci.openshift.org
+    name: kubevirt-builder
+    namespace: ci
+    tag: 30-5.8.1
```

Comment:
where does this kubevirt-builder container come from? would it not make sense for us to build this build container from the upstream sources?

Reply:
I've added an image stream to import the kubevirt builder images from the docker registry: https://github.com/openshift/release/blob/7f2a4c4d470010cc9e05e1dc34f70b2e75b5b357/core-services/supplemental-ci-images/kubevirt_builder_is.yaml

Reply:
AFAIU there are technical reasons why this had to be done in a separate PR.

Reply:
And as you can see [here](https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_release/8211/rehearse-8211-pull-ci-kubevirt-kubevirt-master-images/2/build-log.txt) it succeeds in using the image, but fails when trying to use the new script (contained in the related kubevirt PR)

Reply:
ah, i see. makes sense then.